recipes/langchain/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "langchain" %}
+{% set version = "0.0.136" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain-{{ version }}.tar.gz
+  sha256: 6ab89b1d0d350792fba1cfe5f236f97d3484490b7ed61294156a1ddd8b68f187
+
+build:
+  entry_points:
+    - langchain-server = langchain.server:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.1,<4.0
+    - pydantic >=1.0.0,<2.0.0
+    - sqlalchemy >=1.0.0,<2.0.0
+    - requests >=2.0.0,<3.0.0
+    - pyyaml >=5.4.1
+    - numpy >=1.0.0,<2.0.0
+    - openapi-schema-pydantic >=1.2.0,<2.0.0
```

Comment:
There will be a race condition and the creation of this feedstock may fail b/c openapi-schema-pydantic is not in the channel yet, be sure to restart the CIs when openapi-schema-pydantic lands.